packages/adapter-netlify/index.js

```diff
@@ -143,7 +143,7 @@ async function generate_edge_functions({ builder }) {
 	writeFileSync(`${tmp}/manifest.js`, `export const manifest = ${manifest};\n`);
 
 	/** @type {{ assets: Set<string> }} */
-	const { assets } = (await import(`${tmp}/manifest.js`)).manifest;
+	const { assets } = JSON.parse(manifest);
```

Comment:
We can't use `JSON.parse` because the manifest is JavaScript code.
```suggestion
  // we have to prepend the file:// protocol because Windows doesn't support absolute path imports
	const { assets } = (await import(`file://${tmp}/manifest.js`)).manifest;
```